spec/system/wizard_cache_cleared_spec.rb

```diff
@@ -34,6 +34,8 @@
 
   # check ReasonsForRejectionController for configuration
   scenario 'on entrypoint checks, when the user re-enters a wizard from a specified entrypoint the cache is cleared' do
+    FeatureFlag.deactivate(:structured_reasons_for_rejection_redesign)
```

Comment:
why do we need to deactive here?

Reply:
Line 46 `when_i_choose_to_reject_application` splits off into current/redesigned R4R depending on the flag. Line 47 is opinionated about which reasons form is rendered.